core/nandtool/flashmng.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Phoenix-RTOS
+ *
+ * NANDtool utility
+ *
+ * Flash Server Manager
+ *
+ * Copyright 2019, 2021 Phoenix Systems
+ * Author: Hubert Buczynski, Lukasz Kosinski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <sys/types.h>
+
+#include <imx6ull-flashsrv.h>
```

Comment:
this will compile only for imx6ull, so (unfortunately) we should name it `imx6ull-nandtool` (for now)?

(I believe we need to create common API for all flash servers in the near future...)

Reply:
I'm leaving the `nandtool` app name - significant work has been done for common flash API and soon we'll be able to make this portable across many platforms.